src/app/business/delfin/storages/storages.component.html

```diff
@@ -0,0 +1,528 @@
+                    </p-footer>
+                </p-card>
+            </div>
+        </div>
+        <div class="ui-g-row">
+            <div class="ui-g-6 array-usable-capacity-widget">
+                <p-card  styleClass="ui-card-shadow">
+                    <p-header style="padding: 0.15rem 0.2rem 0 0.2rem;">
+                        <div class="ui-g-row summary-card-header-container" >
+                            <div class="ui-g-6 left-items" style="padding-left: 0.2rem;">
+                                <div class="ui-card-title">Array Usable Capacity Summary</div>
+                                <div class="ui-card-subtitle"><span class="free-storage">Free</span> <span>/</span><span class="used-storage">Used</span> <span>/</span><span class="total-storage">Total</span></div>
+                            </div>
+                            <div class="ui-g-offset-5 ui-g-1">
+                                <button pButton icon="fa fa-fw fa-pie-chart" (click)="showChart()" type="button"></button>
+                            </div>
+                        </div>
+                    </p-header>
+                    <div>
+                        <p-scrollPanel styleClass="storage-summary-panel-2">
+                            <div *ngFor="let device of allStorages" class="single-storage-item">
+                                <div class="ui-g storage-title">
+                                    <div class="ui-g-6">
+                                        <h4> <a [routerLink]="['/storageDetails/' + device.id]"> {{device.name}}</a> </h4>
+                                    </div>
+                                    <div class="ui-g-5 capacity-stats-bar">
+                                        <span class="free-storage">
+                                            {{device.capacity && device.capacity.free ? device.capacity.free : '-'}} 
+                                            
+                                        </span> 
+                                        <span>/</span>
+                                        <span class="used-storage">
+                                            {{device.capacity && device.capacity.used ? device.capacity.used : '-'}} 
+                                            
+                                        </span> 
+                                        <span>/</span>
+                                        <span class="total-storage">{{device.capacity && device.capacity.total ? device.capacity.total : '-'}} 
+                                            
+                                        </span>
+                                    </div>
+                                </div>
+                                <div *ngIf="device.capacity && device.capacity.usage" class="storage-usage-bar" style="padding: 5px 25px 5px 5px;">
+                                    <p-progressBar [ngClass]="{'capacity-usage-normal': device.capacity.usage < 75, 
```

Comment:
Can the soft and hard limit be user configurable?

Reply:
Can be added as a enhancement . At the moment we can keep certain limits. Will take this up in the next set of requirments.
Will raise an issue to track this.

Reply:
Raised an issue to track this. #398